app/views/layouts/_flash.html.erb

```diff
@@ -1,7 +0,0 @@
-<% flash.each do |name, msg| %>
```

Comment:
Because it's erb, i created a haml version, and I'm not sure why it's not showing up here, it's in my local branch and if I do git status nothing shows to be different